src/utils/datoUtils.js

```diff
@@ -316,9 +316,10 @@ export const addWeeks = (date, numberOfWeeks) => {
 
 const ONE_WEEK_MILLIS = 7 * 24 * 60 * 60 * 1000;
 
-export const getWeeksSinceDate = (date) => {
+export const getWeeksSinceDate = (startDate, endDate) => {
```

Comment:
Funksjonsnavnet gir ikke helt mening når man tar inn to datoer her. Mulig man bare kunne brukt `getWeeksBetween` direkte fra komponenten på et vis?

Reply:
👍